modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/BootstrapIndexing.scala

```diff
@@ -0,0 +1,86 @@
+package ch.epfl.bluebrain.nexus.kg.service
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.Uri
+import cats.instances.future._
+import ch.epfl.bluebrain.nexus.commons.sparql.client.SparqlClient
+import ch.epfl.bluebrain.nexus.kg.core.domains.DomainEvent
+import ch.epfl.bluebrain.nexus.kg.core.instances.InstanceEvent
+import ch.epfl.bluebrain.nexus.kg.core.organizations.OrgEvent
+import ch.epfl.bluebrain.nexus.kg.core.schemas.SchemaEvent
+import ch.epfl.bluebrain.nexus.kg.indexing.domains._
+import ch.epfl.bluebrain.nexus.kg.indexing.instances._
+import ch.epfl.bluebrain.nexus.kg.indexing.organizations._
+import ch.epfl.bluebrain.nexus.kg.indexing.schemas._
+import ch.epfl.bluebrain.nexus.kg.service.config.Settings
+import ch.epfl.bluebrain.nexus.service.commons.persistence.SequentialIndexer
+
+import scala.concurrent.{ExecutionContextExecutor, Future}
+
+trait BootstrapIndexing {
```

Comment:
could we have an object like the following instead?

```
object StartIndexers {
  def apply(settings: Settings, sparqlClient: SparqlClient[Future], apiUri: Uri)(implicit as: ActorSystem, ec: ExecutionContextExecutor): Unit = ???
}
```
I think it makes it more readable when used:
```
StartIndexers(settings, sparqlClient, apiUri)
```